src/core/builders/create-form-validator.spec.ts

```diff
@@ -1095,6 +1095,46 @@ describe("createFormValidator", () => {
 
     expect(validation).toEqual([{ field: Schema.string, error: null }]);
   });
+
+  it("should trigger parent validation when child is validating", async () => {
```

Comment:
nice :)

I think we need to test one more thing though.
Letes say we have more complex Schema:
```ts
      objectArray: fields.object({
        arrayString: fields.array(fields.string()),
        arrayNumber: fields.array(fields.number()),
      }),
```

now, when we run validation for `Schema.objectArray.arrayString.nth(0)` we want parent validators to be triggerd, but we do not want sibling validators to be triggered - `arrayNumber` is not affected and does not need validation, right?

You could alos use `jest.fn()` together with `toHaveBeenCalled()` and `not.toHaveBeenCalled()` assertions, maybe that would be easier thant checking actual validation errros